supportedBackends/google/pipelines/batch/src/main/scala/cromwell/backend/google/pipelines/batch/GcpBatchRequest.scala

```diff
@@ -0,0 +1,10 @@
+package cromwell.backend.google.pipelines.batch
+
+import cromwell.backend.google.pipelines.batch.GcpBatchRequestFactory.CreatePipelineParameters
+import cromwell.core.WorkflowId
+
+case class GcpBatchRequest(workflowId: WorkflowId,
```

Comment:
Given that this is used by many classes, it is ideal to make it a generic model instead of being an actor message, hence, I extracted it from the actor code.